### ISSUE TITLE
 val annotations 

### ISSUE THREAD

**Issue by nerminsamet** [youtubevos/MaskTrackRCNN#38]
Hi, 
Are you going to release the annotations for the validation set?
Thanks.

**Comment by linjieyangsc** [comment]
The current validation set serves as a challenge-validation subset and will not be publicly released. Most papers will use this set to report result and compare with others, so we think it is better to have it withheld to avoid tuning models aggressively on it. Feel free to split a subset in the training data for purpose of validation. Thanks.
